sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/ColumnarBatch.java

```diff
@@ -307,64 +293,69 @@ public void update(int ordinal, Object value) {
 
     @Override
     public void setNullAt(int ordinal) {
```

Comment:
just for curiosity, will we really update the rows returned by `ColumnarBatch.rowIterator`?